backend/src/main/java/com/example/backend/service/HomeworkService.java

```diff
@@ -0,0 +1,28 @@
+package com.example.backend.service;
+import com.example.backend.model.Homework;
+import com.example.backend.repository.HomeworkRepository;
+import lombok.RequiredArgsConstructor;
+import org.bson.BsonBinarySubType;
+import org.bson.types.Binary;
+import org.springframework.stereotype.Service;
+import org.springframework.web.multipart.MultipartFile;
+import java.io.IOException;
+
+@Service
+@RequiredArgsConstructor
+public class HomeworkService {
+
+    private final HomeworkRepository photoRepo;
```

Comment:
it's not wrong but confusing to name the HomeworkRepo photoRepo ;)

Reply:
It doesn't seem important, I'll fix it immediately :)